replacer.go

```diff
@@ -115,12 +115,13 @@ func (r *replacer) ReplaceAll(input, empty string) string {
 			}
 		}
 		if !found {
+			lastWriteCursor = i
```

Comment:
Just wondering, which test case does this fix?

Reply:
If you uncomment this fix, the ReplaceAll test fails:
`    replacer_test.go:155: Expected '{teval1{as{{dftest-123' got '{teval1{as{as{{as{{dftest-123' for '{te{test1}{as{{df{1}'
    replacer_test.go:155: Expected 'val1 {nope} test-123 ' got 'val1  {nope} test-123 ' for '{test1} {nope} {1} '`
--> it doesn't count correctly if an invalid placeholder is found. This results for example in the three "{as" and the additional empty space before "{nope}"